repositories/implementations/csv_mlb_player.go

```diff
@@ -87,3 +89,98 @@ func (repo *CSVMLBPlayerRepository) GetMLBPlayerByID(id int) (*e.MLBPlayer, erro
 
 	return nil, nil
 }
+
+func (repo *CSVMLBPlayerRepository) GetMLBPlayerDesired(filterType string, totalItems int, itemsPerWorker int) ([]e.MLBPlayer, error) {
+	f, err := os.Open(repo.filePath)
+
+	if err != nil {
+		log.Println("error opening the file", err)
+		return nil, errors.New("error opening the file")
+	}
+	defer f.Close()
+
+	reader := csv.NewReader(f)
+	reader.Read()
+
+	pCount := 0
+	job := func(jobID int) (*e.MLBPlayer, bool) {
```

Comment:
you would probably like to define this function as a regular function, it takes a lot of space inside of here